src/constraint_system/helper.rs

```diff
@@ -98,7 +103,7 @@ where
     // Compute Commit and Verifier Key
     let (sonic_ck, sonic_vk) = SonicKZG10::<E, DensePolynomial<E::Fr>>::trim(
         &universal_params,
-        verifier.circuit_size().next_power_of_two(),
+        verifier.circuit_size().next_power_of_two() + 7,
```

Comment:
Is that a `+`? 
And if so, shouldn't it be 6?

Reply:
You're right, should be a `+6`

Reply:
fixed